javascriptv3/example_code/bedrock/actions/list-foundation-models.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { fileURLToPath } from "url";
+
+// snippet-start:[javascript.v3.bedrock.actions.ListFoundationModels]
```

Comment:
You can use `snippet_files` in the metadata instead, and leave out the snippet tag here. We prefer `snippet_files` when it makes sense.

Reply:
Done